src/overlays/actors/ovl_player_actor/z_player.c

```diff
@@ -75,15 +75,17 @@ typedef struct SpecialRespawnInfo {
 } SpecialRespawnInfo; // size = 0x10
 
 typedef enum AnimSfxType {
-    /* 1 */ ANIMSFX_TYPE_1 = 1,
-    /* 2 */ ANIMSFX_TYPE_2,
-    /* 3 */ ANIMSFX_TYPE_3,
-    /* 4 */ ANIMSFX_TYPE_4,
-    /* 5 */ ANIMSFX_TYPE_5,
-    /* 6 */ ANIMSFX_TYPE_6,
-    /* 7 */ ANIMSFX_TYPE_7,
-    /* 8 */ ANIMSFX_TYPE_8,
-    /* 9 */ ANIMSFX_TYPE_9
+    /* 1 */ ANIMSFX_TYPE_NO_OFFSET = 1,
+    /* 2 */ ANIMSFX_TYPE_WITH_FLOOR_OFFSET,
+    /* 3 */ ANIMSFX_TYPE_WITH_FLOOR_AND_AGE_OFFSETS,
+    /* 4 */ ANIMSFX_TYPE_WITH_AGE_OFFSET,
+    // the following 4 types ignore the value of sfxId and alwats play a specific sound (taking iron boots into account)
+    /* 5 */ ANIMSFX_TYPE_LANDING,   // landing sound
+    /* 6 */ ANIMSFX_TYPE_WALKING_1, // walking sound (as if link was moving with speed 6)
+    /* 7 */ ANIMSFX_TYPE_JUMPING,   // jumping sound
+    /* 8 */ ANIMSFX_TYPE_WALKING_2, // walking sound (as if link was standing still)
```

Comment:
I can rename them to `ANIMSFX_TYPE_RUNNING` & `ANIMSFX_TYPE_WALKING` but idk what to change the comments to as they are still playing the same sound.
It may also be a good idea to mention the sound enum values (`NA_SE_PL_WALK_GROUND` for example) instead of just general names.

Reply:
staying simple and generic for comments is better, if someone wants the specific sound they can dig in the source
comments should help simplify the system, player is pretty complicated already :)

for the comments maybe
`walking sound (as if link was moving fast)`
`walking sound (as if link was moving slow)`

Reply:
Done